### DISCUSSION TITLE
[News] Future of Zero and 1 is uncertain but should be fine.

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#701] [General]
Announced by Arch Linux Arm on 20211212: [End of life for v5/v6, changes for v7](https://archlinuxarm.org/forum/viewtopic.php?f=3&p=68240&sid=9dd490569ed9a760bbd635ce73623225#p68240)

**Comment by R101G** [comment]
Hi @rern

Just out of curiosity, what is the advantage of using Arch for rAudio rather than a Debian based system?
